roles/StackStorm.st2web/tasks/main.yml

```diff
@@ -66,8 +66,8 @@
   become: yes
   file:
     state: link
-    src: /etc/nginx/sites-available/st2.conf
-    dest: /etc/nginx/sites-enabled/st2.conf
+    src: /etc/nginx/sites-available/"{{ st2web_nginx_conf | basename }}"
```

Comment:
should it always be called st2.conf?

this way the custom one will overwrite the existing if it exists?

Reply:
hm. Maybe? I like having them different personally - but I can see your utility...